### PR TITLE
[452] feat: language switcher and login shortcodes

### PR DESCRIPTION
**Pull request by jremen**
**Changes proposed in this Pull Request**
Language switcher and login shortcodes for Header

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#452

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -36,3 +34,104 @@
 		require_once $filepath;
 	}
 	unset( $file, $filepath );
+
+	function yt_videodetails( $video_id ) {
```
Comment (mvstepanek) [inline functions.php:38]: Do not store everything in `functions.php`. Instead, make structure followed structure in LA and PAP.

**Review by mvstepanek — CHANGES_REQUESTED** [review]

File: functions.php (hunk 1 above)
Comment (jremen) [reply functions.php]: Removed, used for testing content filter

File: lib/shortcodes/login.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<?php
+
+function ms_login() {
```
Comment (mvstepanek) [inline lib/shortcodes/login.php:3]: Login will be entirely in React since all UI of URLsLab will be in React, including the sign in and sign up screen.

**Review by mvstepanek — CHANGES_REQUESTED** [review]

File: lib/shortcodes/login.php (hunk 2 above)
Comment (jremen) [reply lib/shortcodes/login.php]: Removed

**Review by mvstepanek — APPROVED** [review]

**Pull request merged**